### DISCUSSION TITLE
pip install fails perhaps due to manylinux1 end of life

### DISCUSSION THREAD

**Discussion by kenfallon** [openai/whisper#1827] [General]
I am unable to install on `Fedora release 39 (Thirty Nine)` due to missing missing dependencies

```
$ pip install openai-whisper
...
ERROR: Cannot install openai-whisper==20230117, openai-whisper==20230124, [... etc] 
because these package versions have conflicting dependencies
```

This appears to be because the installer uses [manylinux1](https://github.com/pypa/manylinux) but it is not supported on  `Fedora release 39 (Thirty Nine)`

**Comment by jongwook** [comment]
Can you share the full pip error log (maybe using `pip --log ...`) and your Python version? Whisper package itself is [source-only](https://pypi.org/project/openai-whisper/#files) and its dependencies like triton, tiktoken, numba, numpy seem to be available in manylinux2014 targets.

**Comment by hsnfirdaus** [comment]
I'm also using Fedora 39. I think the cause of issue is python version. Fedora 39 ships Python version 3.12.0 by default. Try install [Python 3.9](https://developer.fedoraproject.org/tech/languages/python/multiple-pythons.html). Using venv i'm able to install and run this project.

**Comment by kenfallon** [comment]
Thanks you @jongwook and @hsnfirdaus for your help.

The issue seems related to triton and torch.

I was able to install it using
```
$ python3.9 -m venv ~/tmp/pip3.9/
$ ~/tmp/pip3.9/bin/activate
(pip3.9) $ pip install openai-whisper --log pip-install-openai-whisper-venv-3.9.log
```

[pip-install-openai-whisper.log](https://github.com/openai/whisper/files/13578595/pip-install-openai-whisper.log)
[pip-install-openai-whisper-venv-3.9.log](https://github.com/openai/whisper/files/13578596/pip-install-openai-whisper-venv-3.9.log)
